java/fury-core/src/main/java/io/fury/serializer/ExternalizableSerializer.java

```diff
@@ -41,6 +41,7 @@ public ExternalizableSerializer(Fury fury, Class<T> cls) {
     super(fury, cls);
     try {
       constructor = cls.getConstructor();
+      constructor.setAccessible(true);
```

Comment:
Good catch! I missed this before. Can we use `io.fury.util.ReflectionUtils#getCtrHandle(java.lang.Class<?>, boolean)` instead, since `setAccessible` won't work for JDK17 if the class isn't accessible from this module

Reply:
Oh, I didn't even know. Thanks for pointing that out. Will update the PR.

Reply:
Not sure about the `checked` flag, but I set it to `false`...